packages/eslint-plugin/CHANGELOG.md

```diff
@@ -4,11 +4,18 @@
 
 - The [`@wordpress/no-unused-vars-before-return` rule](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars-before-return.md) has been improved to exempt object destructuring only if destructuring to more than one property.
 - Stricter JSDoc linting using [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc).
+- Stricter validation enabled for test files only using new `test-e2e` and `test-unit` rulesets.
 
 ### New Features
 
 - New Rule: [`@wordpress/no-unguarded-get-range-at`](https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/docs/rules/no-unguarded-get-range-at.md)
 - Enable `wp` global by default in the `recommended` config.
+- New ruleset `test-e2e` added for end-to-end tests validation.
+- New ruleset `test-unit` added for unit tests validation.
+
+### Enhancements
+
+- Remove `@wordpress/dependency-group` and `@wordpress/gutenberg-phase` rules from the `custom` and `recommended` configs and leave them as opt-in features.
```

Comment:
👍 This is mostly a breaking change for consumers that are using these rules.  Most consumers won't want/need them so I like.

Reply:
Do you think it should be highlighted as breaking change? I wasn’t sure where to put it. Given that, there are other breaking changes, we can more freely move it if necessary 😃

Reply:
I'm not sure either.  The scope of when it's considered a breaking change is fairly narrow and it's only breaking in the sense the rules won't be applied automatically anymore.